### PR TITLE
Agenda 11.04.2019

### PR DESCRIPTION
**Pull request by daneos**
Hi all,
Here's the agenda for the upcoming meeting.
Feel free to add your ideas.
@amilan, will you do the presentation on DSConfig?

### CODE REVIEW

File: 20190411/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Sardana Follow-up Meeting - 2019/04/11
+
+To be held on 2019/04/11 at 14:00
+
+## Agenda
+
+1. Presentation
+	* Actual state of DSConfig (?)
+2. Urgent user problems/issues - Round table
+3. Overview of current developments / PR distribution
```
Comment (reszelaz) [inline 20190411/AGENDA.md:10]: I would like to talk about:
* SEP2
* CI for Windows #1078

**Comment by reszelaz** [comment]
For me the agenda looks ok!

> @amilan, will you do the presentation on DSConfig?

It would be great to hear about this. If not this, maybe about the Jupyter and MS PoC done at MaxIV? :) It does not need to be a fancy presentation, just a quick demo and a space for questions would be more then enough!

**Comment by amilan** [comment]
Hi,
The agenda looks OK, and I think that we can keep the presentation for the DSConfig ... maybe I will try to get someone else from MAXIV to make that presentation, as a way to involve more people in these meetings ... we will see...

Cheers,
Antonio.

File: 20190411/AGENDA.md (hunk 1 above)
Comment (daneos) [reply 20190411/AGENDA.md:10]: Added these topics, and also I think we should start working on a roadmap for Python3 support.

**Comment by reszelaz** [comment]
@daneos IMHO the agenda looks pretty ok to be announced to the mailing list. Many thanks!

**Pull request merged**